sunkit_pyvista/mapsequence_animator.py

```diff
@@ -0,0 +1,27 @@
+import time
```

Comment:
Should the file be called `mapsequence_animator.py`?

Reply:
I'm not sure, I have created an animator which specifically performs only this function